README.md

```diff
@@ -38,7 +44,7 @@ $ npm start
 
 ## Authors
 
-👤 **Author1**
+👤 **Author**
```

Comment:
- [ ] It would be better if we add the name of the designer we took the design inspiration from in our Acknowledgment section of the README file